bin/build-decoder.sh

```diff
@@ -10,6 +10,7 @@ cd "$deps/cache-decoder"
 if [ -z "$SKIP_BUILD" ]; then
   yarn install --frozen-lockfile
   yarn build
+  mkdir -p "$pwd/site"
   mv dist ../../site/cache-decoder
```

Comment:
```suggestion
   mv dist $pwd/docs/cache-decoder
```
you probably forgot that?

Reply:
It was suprise to me we use both `docs` and `site` folders

Reply:
docs is the mkdocs source folder to build from. So if we copy the decoder there, it will be automatically copied to site, which is the final build folder of mkdocs

Reply:
I see now. For some reason, I concluded at some first stage of my work that only Markdown files are transformed/copied from `docs` to `site`. Now I'm adjusting everything to make it work as you proposed.